src/pages/index.js

```diff
@@ -163,14 +153,17 @@ function handleHeartButton(item) {
     api
       .likeCard(item.getId())
       .then((respond) => {
+        console.log(respond);
         item.setLikeStatus(respond.isLiked);
       })
       .catch((err) => {
+        console.error("Error:", err);
       });
   } else {
     api
       .unlikeCard(item.getId())
       .then((respond) => {
+        console.log(respond)
         item.setLikeStatus(respond.isLiked);
         item.isLiked = newIsLikedStatus;
```

Comment:
`item.isLiked = newIsLikedStatus;` is not needed anymore because you set the color inside `setLikeStatus` already